Automattic/wp-calypso
# Jetpack Pricing | Add amount saved in bundles to featured item card

#### Proposed Changes

* This PR adds amount saved text to featured item cards for bundles on Jetpack pricing page.

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-pricing/featured/amount-saved`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Go to Bundles tab
- Confirm that you see "Save {{amount}}/mo vs buying individually" text inside featured card
- Hover over it
- Confirm that you see a tooltip that says, `For details, click on "More about {{product}}"`
- Confirm that the design matches figma

#### Pre-merge Checklist





Completes: 0-as-1203463248826787/f

<img width="1260" alt="Screenshot 2022-12-06 at 10 49 19 AM" src="https://user-images.githubusercontent.com/18226415/205823358-e505b6d9-2d59-40b0-a8c2-31619474de25.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203463248826787
